connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -4454,6 +4486,106 @@ object functions {
    */
   def right(str: Column, len: Column): Column = Column.fn("right", str, len)
 
+  /**
+   * Masks the given string value. The function replaces characters with 'X' or 'x', and numbers
+   * with 'n'. This can be useful for creating copies of tables with sensitive information
+   * removed.
+   *
+   * @param input
+   *   string value to mask. Supported types: STRING, VARCHAR, CHAR
+   * @group string_funcs
+   * @since 3.5.0
+   */
+  def mask(input: Column): Column = Column.fn("mask", input)
+
+  // scalastyle:off line.size.limit
```

Comment:
I think we should fix the code style and remove these `scalastyle:off line.size.limit`

Reply:
+1, please remove them,

I know there are some usage of `scalastyle:off line.size.limit` in this file, but they are due to long hyperlink string which can not be broken into multi line.

In this PR, i think it is not needed

Reply:
Done. thanks @LuciferYang @zhengruifeng